Python/Examples/src/plugins/Root.cpp

```diff
@@ -0,0 +1,256 @@
+#include "ActsExamples/Io/Root/RootPropagationSummaryWriter.hpp"
+#include "ActsExamples/Io/Root/RootSeedWriter.hpp"
+#include "ActsExamples/Io/Root/RootSimHitReader.hpp"
+#include "ActsExamples/Io/Root/RootSimHitWriter.hpp"
+#include "ActsExamples/Io/Root/RootSpacepointWriter.hpp"
+#include "ActsExamples/Io/Root/RootTrackFinderNTupleWriter.hpp"
+#include "ActsExamples/Io/Root/RootTrackFinderPerformanceWriter.hpp"
+#include "ActsExamples/Io/Root/RootTrackFitterPerformanceWriter.hpp"
+#include "ActsExamples/Io/Root/RootTrackParameterWriter.hpp"
+#include "ActsExamples/Io/Root/RootTrackStatesWriter.hpp"
+#include "ActsExamples/Io/Root/RootTrackSummaryReader.hpp"
+#include "ActsExamples/Io/Root/RootTrackSummaryWriter.hpp"
+#include "ActsExamples/Io/Root/RootVertexNTupleWriter.hpp"
+#include "ActsExamples/Io/Root/RootVertexReader.hpp"
+#include "ActsExamples/Io/Root/RootVertexWriter.hpp"
+#include "ActsPython/Utilities/Helpers.hpp"
+#include "ActsPython/Utilities/Macros.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/stl/filesystem.h>
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+using namespace Acts;
+using namespace ActsExamples;
+using namespace ActsPython;
+
+PYBIND11_MODULE(ActsExamplesPythonBindingsRoot, root) {
+  // Input
+  {
+    ACTS_PYTHON_DECLARE_READER(RootParticleReader, root, "RootParticleReader",
+                               outputParticles, treeName, filePath);
+
+    ACTS_PYTHON_DECLARE_READER(RootVertexReader, root, "RootVertexReader",
+                               outputVertices, treeName, filePath);
+
+    ACTS_PYTHON_DECLARE_READER(
+        RootMaterialTrackReader, root, "RootMaterialTrackReader",
+        outputMaterialTracks, treeName, fileList, readCachedSurfaceInformation);
+
+    ACTS_PYTHON_DECLARE_READER(RootTrackSummaryReader, root,
+                               "RootTrackSummaryReader", outputTracks,
+                               outputParticles, treeName, filePath);
+    ACTS_PYTHON_DECLARE_READER(RootMuonSpacePointReader, root,
+                               "RootMuonSpacePointReader", outputSpacePoints,
+                               filePath, treeName);
+
+    ACTS_PYTHON_DECLARE_READER(
+        RootAthenaNTupleReader, root, "RootAthenaNTupleReader", inputTreeName,
+        inputFilePath, outputTrackParameters, outputTruthVtxParameters,
+        outputRecoVtxParameters, outputBeamspotConstraint);
+
+    ACTS_PYTHON_DECLARE_READER(
+        RootAthenaDumpReader, root, "RootAthenaDumpReader", treename,
+        inputfiles, outputMeasurements, outputPixelSpacePoints,
+        outputStripSpacePoints, outputSpacePoints, outputClusters,
+        outputMeasurementParticlesMap, outputParticleMeasurementsMap,
+        outputParticles, onlySpacepoints, onlyPassedParticles,
+        skipOverlapSPsPhi, skipOverlapSPsEta, geometryIdMap, trackingGeometry,
+        absBoundaryTolerance, onlySpacepoints, noTruth, readCellData);
+
+#ifdef WITH_GEOMODEL_PLUGIN
```

Comment:
Did we have this before already? This is a bit awkward.

Reply:
Just checked - it is also in main/head.